models/conditional_grants.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "conditional_grants_facts",
     "label": "Conditional Grants",
-    "description": "This is based on the return forms for individual grants.",
+    "description": "Details of grants received from national government.",
+    "return_forms": ["Individual Grants"],
```

Comment:
I don't think Individual Grants is a _term_. I was referring to each grant having an individual return form. See secrion _Conditional Grant Returns_ at https://mfmamirror.github.io/Return_Forms/